tests/unit/specs/game-state.spec.js

```diff
@@ -0,0 +1,20 @@
+import { describe, it , expect} from 'vitest';
+import { fixture } from '../fixtures/gameState';
+
+describe('Coverting GameState\'s and GameStateRow\'s', () => {
+
+  it('Converts GameState to GameStateRow (packing)', async () => {
+    const gameStateRow = await sails.helpers.gamestate.saveGamestate(fixture.gameState);
+
+    //remove attributes added while creating the entry in the database
+    const attributesToRemove = ['createdAt', 'id', 'updatedAt'];
+    attributesToRemove.forEach(el => delete gameStateRow[el]);
+
+    expect(fixture.gameStateRow).toEqual(gameStateRow);
+  });
```

Comment:
Since we're testing saveGameState, let's confirm that it persists the record as expected in the db as well. This will also require wiping the db between tests, which is a good practice, regardless.

Also as a note, we put the 'actual' value we are testing as the arg to `expect()`, then the value we want it to equal as the arg to `toEqual()` i.e. `expect(variableIAmTesting).toEqual(valueIWantItToHave). ([docs](https://vitest.dev/api/expect.html))
```suggestion
  beforeEach(async () => {
    await sails.helpers.wipeDatabase();
  });

  it('Converts GameState to GameStateRow (packing)', async () => {
    const gameStateRow = await sails.helpers.gamestate.saveGamestate(fixture.gameState);

    //remove attributes added while creating the entry in the database
    const stripDbAttributes = (obj) => {
      const attributesToRemove = ['createdAt', 'id', 'updatedAt'];
      attributesToRemove.forEach(attr => delete obj[attr]);      
    };

    stripDbAttributes(gameStateRow);
    expect(gameStateRow).toEqual(fixture.gameStateRow);

    const dbGameStateRows = GameState.find({});
    expect(dbGameSTateRows.length).toEqual(1);
    const [ savedRow ] = dbGameStateRows;
    stripDbAttributes(savedRow);
    expect(savedRow).toEqual(fixture.gameStateRow);
  });
```